ci-operator/config/openshift/cluster-ingress-operator/openshift-cluster-ingress-operator-master.yaml

```diff
@@ -38,3 +38,7 @@ tests:
     defaultMode set' run-tests
   openshift_installer:
     cluster_profile: aws
+- as: e2e-aws-router
```

Comment:
I think we should just modify the existing `aws-e2e` job in here?

Reply:
I didn't want to touch that one. I'll change it